packages/provider-elasticsearch/src/example-types/results/highlighting.test.js

```diff
@@ -394,3 +396,55 @@ describe('containsHighlightTagRegex()', () => {
     expect(regex.test(text)).toEqual(false)
   })
 })
+
+describe('Highlight field aggregation', () => {
```

Comment:
We should name the describe block after the fn is being tested. I added the article that I mostly follow when writing unit tests in notion: https://www.notion.so/Best-Practices-0d931662db624a77af8dd09cc41e01bf